uisdk/src/main/java/com/karhoo/uisdk/screen/rides/detail/RideDetailMVP.kt

```diff
@@ -36,7 +36,7 @@ interface RideDetailMVP {
 
         fun addTripInfoObserver(tripInfoListener: OnTripInfoChangedListener?)
 
-        fun checkCancellationSLA(tripStatus: TripStatus, serviceCancellation: ServiceCancellation?, context: Context)
+        fun checkCancellationSLA(trip: TripInfo, serviceCancellation: ServiceCancellation?, context: Context)
```

Comment:
Perhaps a bit of personal preference but can we change the signature of this method to
`fun checkCancellationSLA(context: Context, trip: TripInfo, serviceCancellation: ServiceCancellation?)`

it is better to keep consistency in our methods to have the context as the first parameter